gradle/libs.versions.toml

```diff
@@ -1,5 +1,7 @@
 [versions]
+agp = "8.10.0"
 compose = "1.8.2"
+composeUi = "1.8.3"
```

Comment:
If we want to keep this updated by @renovate-bot, we need to add a dependency which refs it, like https://github.com/ComposeGears/Valkyrie/blob/afced72a3048cb5aac7185715da214a63ab5f5e2/gradle/libs.versions.toml#L30-L30

Reply:
Generally, I don't get an idea why we add compose dependency by gradle plugin?

Reply:
Yeah, we have no need to add `androidx.compose.ui:ui`, users should ensure this is added in their dependencies. Also, we use different versions here, which may break users' build logic.

Reply:
Kept the reference in the toml file so it auto-updates, but it's now only referenced in unit tests